### PR TITLE
feat: Add email to users and allow instrument scientist to see comments on proposal

### PR DESCRIPTION
**Pull request by bolmsten**
See:

https://github.com/UserOfficeProject/user-office-frontend/pull/951

### CODE REVIEW

File: src/models/User.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export class BasicUserDetails {
     public organisation: string,
     public position: string,
     public created: Date,
-    public placeholder: boolean
+    public placeholder: boolean,
+    public email: string
```
Comment (martin-trajanovski) [inline src/models/User.ts:93]: Exposing the email here in the BasicUserDetails might not be GDPR compliant. What do you think about that @bolmsten?

**Review by martin-trajanovski — COMMENTED** [review]
Just a little concerned about the GDPR because I know this data for basic user details can be accessed by anyone I think. Even if you are just a normal user you can query that and get all other users emails.

File: src/models/User.ts (hunk 1 above)
Comment (bolmsten) [reply src/models/User.ts:94]: it is now restricted to only user officer and instrument scientist

**Review by jekabs-karklins — APPROVED** [review]
Looks good

**Pull request merged**